src/entities/entities_test.rs

```diff
@@ -0,0 +1,346 @@
+#[cfg(test)]
+mod tests {
+
+    use crate::entities::{Pair, Route, Trade};
+    use alloy_primitives::address;
+    use once_cell::sync::Lazy;
+    use uniswap_sdk_core::prelude::*;
+
+    const CHAIN_ID: u64 = 3;
+
+    static WETH9: Lazy<Token> = Lazy::new(|| {
+        Token::new(
+            CHAIN_ID,
+            address!("C02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"),
+            18,
+            Some("WETH".to_string()),
+            Some("Wrapped Ether".to_string()),
+            None,
+            None,
+        )
+    });
+    static DECIMAL_PERMUTATIONS: [[u8; 3]; 3] = [[0, 0, 0], [0, 9, 18], [18, 18, 18]];
+
+    fn decimalize(amount: u128, decimals: u8) -> BigInt {
+        BigInt::from(amount) * BigInt::from(10).pow(decimals as u32)
+    }
+
+    #[test]
+    fn test_entities() {
```

Comment:
Why don't you use modules and functions to scope unit tests? How is this a unit test?